challenges/2021-11-16-data-grouping/solution.js

```diff
@@ -0,0 +1,73 @@
+/* 
```

Comment:
We probably don't want this file in the main challenge folder, since it's not pertinent to the challenge setup in Python (the original language).

Reply:
Thanks, see commit below deleting it.